mobile-app/app/screens/WalletNavigator/WalletNavigator.tsx

```diff
@@ -253,17 +259,55 @@ export function WalletNavigator (): JSX.Element {
         }}
       >
         <WalletStackV2.Screen
-          component={Onboarding}
+          component={OnboardingV2}
           name='OnboardingV2'
           options={{
             headerShown: false
           }}
         />
-        <WalletStackV2.Screen
-          component={CreateWalletGuidelines}
+
+        <WalletStack.Screen
+          component={CreateWalletGuidelinesV2}
           name='CreateWalletGuidelines'
           options={{
-            headerTitle: translate('screens/WalletNavigator', 'Guidelines')
+            headerTitle: translate('screens/WalletNavigator', 'New Wallet')
+          }}
+        />
+
+        <WalletStack.Screen
+          component={CreateMnemonicWalletV2}
+          name='CreateMnemonicWallet'
+          options={{
+            headerTitle: translate('screens/WalletNavigator', 'View Recovery Words'),
+            headerRightContainerStyle: tailwind('px-2 py-2')
+          }}
+        />
+
+        <WalletStack.Screen
+          component={VerifyMnemonicWalletV2}
+          name='VerifyMnemonicWallet'
+          options={{
+            headerTitle: translate('screens/WalletNavigator', 'Verify Words')
+          }}
+        />
+        <WalletStack.Screen
+          component={PinCreationV2}
+          name='PinCreation'
+          options={{
+            headerTitle: translate('screens/WalletNavigator', 'Create Passcode')
+          }}
+        />
+        <WalletStack.Screen
+          component={PinConfirmationV2}
+          name='PinConfirmation'
+          options={{
+            headerTitle: () => (
+              <HeaderTitle
```

Comment:
let's not use this component since it has v1 UI, just assign the normal screen title like pin creation screen v2

Reply:
PinConfirmation V2 is done and added.

Reply:
what i meant is dont use `<HeaderTitle/>`, just assign `headerTitle: ...`, because `<HeaderTitle/>` will display this 
<img width="176" alt="image" src="https://user-images.githubusercontent.com/40191153/176341930-5ff3bc90-f94a-4b75-a108-0168ade4731b.png">
which is not our v2 UI